tests/unit/test_retry.py

```diff
@@ -0,0 +1,164 @@
+import logging
+from datetime import datetime, timedelta
+from os import getenv
+from typing import AsyncIterator
+from unittest.mock import AsyncMock, MagicMock
+
+import pytest
+
+import jbi.runner
+from jbi.bugzilla.models import Bug, WebhookEvent, WebhookRequest
+from jbi.queue import QueueItem, get_dl_queue
+from jbi.retry import retry_failed
+
+RETRY_TIMEOUT_DAYS = getenv("RETRY_TIMEOUT_DAYS", 7)
+
+
+@pytest.fixture()
+def logger():
```

Comment:
Pytest's [caplog](https://docs.pytest.org/en/6.2.x/logging.html#caplog-fixture) fixture could be useful here

Reply:
Oh yeah that's much easier to work with